src/content/home-page.json

```diff
@@ -18,17 +18,12 @@
 			"ctaText": "Try cloud for free"
 		}
 	},
-	"learnSection": {
+	"certificationsSection": {
```

Comment:
I've worked off the existing `learnSection` content, updated heading, description, and link text to match mockups. Have left the collection cards as-is.